### ISSUE TITLE
reproduce result on mini-imagenet

### ISSUE THREAD

**Issue by kailigo** [dragen1860/MAML-Pytorch#7]
Thanks for sharing the code.
I am trying to reproduce the result you report on the mini-imagenet by running 
python miniimagenet_train.py

The training log shows that the accuracy stops at around 0.30. Any clues for this problem? 
By the way, is it possible for you to share your training logs? 
Much appreciated for your help.

**Comment by dragen1860** [comment]
Hi, Maybe you can comment the line: `if training:` 
https://github.com/dragen1860/MAML-Pytorch/blob/69c08a02b953dcfc3205962f42985f751afb5057/maml.py#L399

**Comment by kailigo** [comment]
Thanks for the reply. I also ran the original Tensorflow version provided by the authors. The results seems fine. Maybe we can check step by step by comparing the intermediate results generated. Please update the code if you find the bug. Thanks for your effort.
